geop/src/main/scala/com/azavea/usace/programanalysis/geop/LayerReader.scala

```diff
@@ -0,0 +1,55 @@
+package com.azavea.usace.programanalysis.geop
+
+import geotrellis.raster.Tile
+import geotrellis.spark.io._
+import geotrellis.spark.{SpatialKey, TileLayerMetadata, _}
+import geotrellis.spark.io.s3.{S3AttributeStore, S3ValueReader}
+import geotrellis.vector.{Extent, MultiPolygon}
+
+import org.apache.spark.SparkContext
+
+object LayerReader {
+  /**
+    * Given a list of layer ids, a multipolygon and a spark context, returns
+    * a list of [[TileLayerRDD]]s cropped to the multipolygon.
+    *
+    * @param    rasterLayerIds  List of [[LayerId]]s to crop
+    * @param    multiPolygon    [[MultiPolygon]] to crop to
+    * @param    sc              [[SparkContext]] for the [[S3LayerReader]]
+    * @return                   List of [[TileLayerRDD]]s which includes only
+    *                           the tiles that either completely or partially
+    *                           overlap given multiPolygon.
+    */
+  def apply(
+    zoom: Int,
+    x: Int,
+    y: Int,
+    sc: SparkContext
+  ): Tile = {
+    val sKey: SpatialKey = SpatialKey(x, y)
```

Comment:
Don't really need to specify the data type as `SpatialKey` here on the left hand side.